src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Startlist/StartlistTimer.razor

```diff
@@ -8,11 +9,11 @@
 
```

Comment:
Why not use `MudText` with `Color` instead of injecting the theme and manually selecting colors using styles?